app/javascript/components/public_health/PatientsTable.js

```diff
@@ -613,7 +624,8 @@ class PatientsTable extends React.Component {
               <Row>
                 <Col md="18">
                   <div id="tab-description" className="lead mt-1 mb-3">
```

Comment:
I can't comment above but you'll have to add the tabs breakpoint styling to `dashboard.scss` for the global dashboard

Reply:
https://github.com/SaraAlert/SaraAlert/pull/937/commits/6f8d9f1349047c6ee2f033f003f86916013aaddf#diff-10d5769ef07c9ffcee6a0a046fbdd42f530f08c90f76ba95c6304f079052713c